pkg/isbsvc/jetstream_service.go

```diff
@@ -128,7 +127,6 @@ func (jss *jetStreamSvc) CreateBuffers(ctx context.Context, buffers []dfv1.Buffe
 			}
 			if _, err := js.CreateKeyValue(&nats.KeyValueConfig{
 				Bucket:       otBucket,
-				Description:  fmt.Sprintf("Offset timeline bucket of buffer [%s]", buffer.Name),
```

Comment:
The description makes each of the table row in the nats jetstream console wrap, which is hard to debug.